proposals/1.0.0/packedMod.md

```diff
@@ -0,0 +1,80 @@
+Standard for packed CrossCode mods. This should simplify sharing and installation of mods.
+
+# Current Situation / Problems
+
+Mods have to be unzip by the user and installed my manually moving them into the mods folder. "Packed mods" are not standartized (rar, 7zip, tar.gz, ...). Not every OS supports every format.
+
+# Solution
+
+Standard for packed mods
+
+* zip of a mod with the extension `.ccmod` (<https://www.iana.org/assignments/media-types/application/zip>)
+* requires package.json in the main directory (see unpacked mods)
+
+## Advantage
+
+* Simplified installation (file association)
+* Standardised container
+* Easy purpose recognition after downloads (`What was this for?`)
+
+## Disadvantage
+
+* More dependencies (if the optional part is implemented)
+* System/OS dependend parts (if the optional part is implemented)
+* Difficulties in testing increase (if the optional part is implemented)
+* Higher programming costs
+
+# Requirement
+
+* Library to unzip the `ccmod` file (e.g. `zlib` - <https://nodejs.org/api/zlib.html>)
+* fs (read `package.json`)
+* fs (move folder)
+
+## Optional
+
+* Library to access windows regestry (e.g. `winreg` - <https://www.npmjs.com/package/winreg>)
+* Tool packed as .app (macOS specific)
+* fs (file edit)
+
+# File handling
+
+Possible installation methods:
+
+## Added manually
```

Comment:
What does this title mean, why is it required?

Reply:
Would be minimum requirement better? There has to be a method to add packed mods. Since "Added by file association" is optional. The method to add mods manually is a requirement. In an CLI this could be done by using the path (`add path/to/mod.ccmod`) or if the tool is using a GUI this can be done drag and drop, path input or moving the .ccmod package to a specific location. This could be the mods folder.

Reply:
It is fine to mark this as required, but it should come with dome sort of explanation as to why this is required. Also, the reader needs some context to what is going on (see my comment on the title `File handling`).

Reply:
Other question: Would modloaders directly loading ccmod files also count as "manual"?

Reply:
This would also count as "manual". `Added automatic or manually` would be a better heading.